dell/xps/15-9500/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, ... }:
+let
+  thermald-conf = ./thermald-conf.xml;
+in
+{
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+  ];
+
+  # Boot loader
+  boot.kernelParams = lib.mkDefault [ "acpi_rev_override" ];
+
+  # This will save you money and possibly your life!
+  services.thermald.enable = lib.mkDefault true;
+
+  # Thermald doesn't have a default config for the 9500 yet, the one in this repo
+  # was generated with dptfxtract-static (https://github.com/intel/dptfxtract)
+  services.thermald.configFile = lib.mkDefault thermald-conf;
+
+  # Set the tlp config to powersave explictly. TLP is enabled in common/pc/laptop.
+  services.tlp.extraConfig = lib.mkDefault "CPU_SCALING_GOVERNOR_ON_AC=powersave\nCPU_SCALING_GOVERNOR_ON_BAT=powersave";
```

This can be now removed given that the fix will be made in nixpkgs itself.

```suggestion
  # Set the tlp config to powersave explictly. TLP is enabled in common/pc/laptop.
  services.tlp.extraConfig = lib.mkDefault "CPU_SCALING_GOVERNOR_ON_AC=powersave\nCPU_SCALING_GOVERNOR_ON_BAT=powersave";
```

`nixos-config-generate` will set the governor to powersave already.